include/vicon_transformer/types.hpp

```diff
@@ -41,7 +42,7 @@ struct SubjectData
     Transformation global_pose;
 
     //! Quality measure of the pose estimation.
-    double quality;
+    double quality = 0.0;
```

Comment:
more info would be nice for 'quality' (what range of values, pro and cons of putting low or high values)

Reply:
Indeed.  Unfortunately, I still have to figure this out myself.  Vicon's documentation is unfortunately not very elaborate on this.  I'm not even sure what the range is.  A code snippet in the documentation suggests its [0.0, 1.0], but I often see values > 1 when running the system...